From a98b8bd41e9e9753f1cd1b93d9276cbd45e3998c Mon Sep 17 00:00:00 2001
From: Prokhar Bhowal <Prokhar.Bhowal@amd.com>
Date: Mon, 6 Jun 2016 17:29:25 -0400
Subject: [PATCH 0533/1722] drm/amd/dal: Fix max pixel clock for DCE11

Corrected max pixel clock to 594 MHz which was incorrectly set to 600 MHz

Signed-off-by: Prokhar Bhowal <Prokhar.Bhowal@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/dce110/dce110_link_encoder.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_link_encoder.c b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_link_encoder.c
index 7f2370d..5af451a 100644
--- a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_link_encoder.c
+++ b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_link_encoder.c
@@ -48,7 +48,7 @@
 /* For current ASICs pixel clock - 600MHz */
 #define MAX_ENCODER_CLK 600000
 
-#define DCE11_UNIPHY_MAX_PIXEL_CLK_IN_KHZ 600000
+#define DCE11_UNIPHY_MAX_PIXEL_CLK_IN_KHZ 594000
 
 #define DEFAULT_AUX_MAX_DATA_SIZE 16
 #define AUX_MAX_DEFER_WRITE_RETRY 20
-- 
2.7.4

